nntrainer/layers/layer_node.h

```diff
@@ -441,6 +458,21 @@ class LayerNode final : public ml::train::Layer, public GraphNode {
     }
   }
 
+  /**
+   * @brief Get the Weight object
+   *
+   * @param idx Identifier of the weight
+   * @return Tensor& Reference to the weight tensor
```

Comment:
return is weight?

Reply:
Are we opening weight? or is it temporary?

Reply:
This is just a wrapping interface, which takes already available information for weight and creates a wrapper for weight.
Further, this opens weights for internal use only. 
This is primarily added for unittest_models.

Reply:
I see. I guess, in the long term it'd better refactor unittest_models later

Reply:
If this function is not used anywhere else after layer_v2 refactor, the implementation here can move the unittest_models itself.